cleancode/src/main/java/com/epam/cleancode/singleresponsibility/Calculator.java

```diff
@@ -1,31 +1,43 @@
 package com.epam.cleancode.singleresponsibility;
 
-public class Calculator{
+public class Calculator {
 
     private static final String SEPARATOR = ",";
 
     public String add(String numbers) {
         validate(numbers);
-        return "sum: " + String.valueOf(getSum(numbers));
+
+        return "summa: " + getSum(numbers);
     }
 
     public String multiply(String numbers) {
         validate(numbers);
-        return "product: " + String.valueOf(getProduct(numbers));
+
+        return "product: " + getProduct(numbers);
     }
 
     private int getSum(String numbers) {
-        int sum = 0;
-        for (String s : numbers.split(SEPARATOR)) if (isNotEmpty(s)) sum += Integer.valueOf(s);
-        return sum;
+        int summa = 0;
+
+        for (String s : numbers.split(SEPARATOR)) {
```

Comment:
please rename 's' to something more readable

Reply:
> for (String number : numbers.split(SEPARATOR)) {
fix https://github.com/TashaB1/EpamTraining2019/commit/e2848c1f5ec0ec7697069142242c0daa4c532733